tamr_unify_client/models/project/resource.py

```diff
@@ -119,6 +122,15 @@ def input_datasets(self):
         alias = self.api_path + "/inputDatasets"
         return DatasetCollection(self.client, alias)
 
+    def attribute_configurations(self):
+        """ Project's attribute's configurations.
+        :returns: the configurations of the attributes of a project
+        :rtype :class: '~tamr_unify_client.models.attribute_configuration.collection.AttributeConfigurationCollection'
```

Comment:
Needs backticks instead of quotes (:class:\`~tamr_unify_client.models.attribute_configuration.collection.AttributeConfigurationCollection\`)